pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -385,6 +385,12 @@ public ServerType getServerType() {
       public static final String CLIENT_SECRET = "clientSecret";
     }
 
+    public static class GcsSegmentOperations {
```

Comment:
Let's move these constants to the GCS module. I want to break the current pattern.